frontend/src/components/RFI/NMF.vue

```diff
@@ -199,7 +174,6 @@ export default {
     '$route.params.urlGuid': {
       async handler() {
         let ccfriId = this.$route.params.urlGuid;
-        console.log('ccfriId = ', ccfriId);
         await this.loadNmf(ccfriId);
```

Comment:
This is old code, but do we need to have the variable ccfriId? Or can we just use the urlGuid directly as a parameter to loadNmf?

Reply:
agreed! it's updated